test/e2e/apps/replica_set.go

```diff
@@ -102,7 +102,14 @@ var _ = SIGDescribe("ReplicaSet", func() {
 		testReplicaSetServeImageOrFail(f, "private", privateimage.GetE2EImage())
 	})
 
-	ginkgo.It("should surface a failure condition on a common issue like exceeded quota", func() {
+	/*
+			    Release : v1.16
```

Comment:
Please reformat this comments.

Reply:
The formatting has been updated. Hopefully it's better :smiley: